service.py

```diff
@@ -68,29 +70,32 @@ def run(self):
 
 if __name__ == "__main__":
 
-    LOG.warn("-->[ service ]")
-    LOG.warn("Delay startup by %s seconds.", DELAY)
+    LOG.info("-->[ service ]")
+    LOG.info("Delay startup by %s seconds.", DELAY)
+
+    if not xbmcvfs.exists(ADDON_DATA):
+        xbmcvfs.mkdirs(ADDON_DATA)
 
     while True:
 
-        if not settings('enableAddon.bool'):
-            LOG.warn("Jellyfin for Kodi is not enabled.")
+        if settings('enableAddon.bool'):
```

Comment:
Why invert the if? It seems only the indentation level has been increased, so I don't think it was necessary